src/views/CourtOrder.vue

```diff
@@ -595,6 +595,7 @@ export default class CourtOrderView extends Mixins(DateMixin, FilingMixin, Commo
       })
 
     this.saving = false
+    this.haveChanges = false
```

Comment:
If creating the filing fails then this flag is cleared, allowing the user to exit this page without actually saving and without a warning. Can this be moved somewhere better?

Reply:
Fixed, thanks.